logprep/processor/dissector/rule.py

```diff
@@ -75,20 +75,23 @@
 from attrs import define, validators, field, Factory
 
 from logprep.filter.expression.filter_expression import FilterExpression
-from logprep.processor.base.rule import Rule
+from logprep.processor.base.rule import SourceTargetRule
 from logprep.util.helper import append, add_and_overwrite
 
 DISSECT = r"(%\{[+&?]?[^%{]*\})"
 SEPARATOR = r"((?!%\{.*\}).+)"
 
 
-class DissectorRule(Rule):
+class DissectorRule(SourceTargetRule):
     """dissector rule"""
 
     @define(kw_only=True)
-    class Config(Rule.Config):
+    class Config(SourceTargetRule.Config):
         """Config for Dissector"""
 
+        source_fields: list = field(factory=list)
+        target_field: str = field(default="")
```

Comment:
no, because in the parent class they are not optional. I only need them in this processor to implement the delete_source_fields behavior. But they should not be needed in the configuration of this processor. (see post_init)